src/app/index-wrapper.jsx

```diff
@@ -9,34 +9,15 @@ import { getAngularService } from 'services/angular-react-helper';
 function IndexWrapper() {
   const $rootScope = getAngularService('$rootScope');
   const Idle = getAngularService('Idle');
-  const Keepalive = getAngularService('Keepalive');
   const authService = getAngularService('authService');
   const networkService = getAngularService('networkService');
-  const $localStorage = getAngularService('$localStorage');
-
-  const isCurrentUserChplAuthenticated = $localStorage.currentUser && $localStorage.currentUser.userId;
 
   useEffect(() => {
-    let deregisterKeepalive;
     if (authService.hasAnyRole(['chpl-admin', 'chpl-onc', 'chpl-onc-acb', 'chpl-cms-staff', 'chpl-developer'])) {
       Idle.watch();
       console.log('Starting Idle in IndexWrapper');
     }
-    if (isCurrentUserChplAuthenticated) {
-      deregisterKeepalive = $rootScope.$on('Keepalive', () => {
-        console.info('Keepalive');
-        if (authService.hasAnyRole(['chpl-admin', 'chpl-onc', 'chpl-onc-acb', 'chpl-cms-staff', 'chpl-developer'])) {
-          networkService.keepalive()
-            .then((response) => {
-              authService.saveToken(response.token);
-            });
-        } else {
-          Idle.unwatch();
-        }
-      });
-    }
-    return deregisterKeepalive;
-  }, [$rootScope, Idle, Keepalive, authService, networkService]);
+  }, [$rootScope, Idle, authService, networkService]);
```

Comment:
```suggestion
  }, [Idle, authService]);
```

These look like the only services that are still in this function. The other ones should be removed as well